python/python/tests/test_dataset.py

```diff
@@ -504,14 +505,13 @@ def test_cleanup_old_versions(tmp_path):
     # Ok, can accept timedelta
     dataset.cleanup_old_versions(older_than=timedelta(days=14))
 
-    print(tmp_path)
-    for root, dirnames, filenames in os.walk(tmp_path):
-        for filename in filenames:
-            print(root + "/" + filename)
+    # print(tmp_path)
```

Comment:
Delete them ?

Reply:
I assume they are useful for validating and non trivial to rewrite so that’s why I kept these lines